### DISCUSSION TITLE
Allow passing options to useResizeObserver / useElementSize hooks to specify the box model to observe

### DISCUSSION THREAD

**Discussion by moimael** [mantinedev/mantine#5754] [Feature requests]
Hi,

`useResizeObserver` / `useElementSize` are useful hooks, but they would be even more useful if we had a way to pass a parameter allowing us to select which box model to observe. 

I'd like to get the size of an element including padding and borders and I'd have to write a custom hook for this instead of being able to use the existing mantine hook. Since `ResizeObserver` supports this feature natively, it'd just be a matter of allowing to pass an option to the hook to enable this feature. 

https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver/observe#options

Thanks!

**Comment by rtivital** [comment]
It will be added in one of the next releases (if it is possible). If you are interested, you are welcome to submit a PR.
